wallet/res/values-bg/strings.xml

```diff
@@ -296,7 +295,7 @@
     <string name="notification_coins_received_msg">Получен %s</string>
     <string name="notification_peers_connected_msg">%d пиъри са свързани</string>
     <string name="notification_inactivity_title">Все още имате Dash на това устройство!</string>
-        <string name="notification_inactivity_message">Все още имате Dash на това устройство! Моля, не забравяйте да запишете фразата за възстановяване, преди да деинсталирате Dash Wallet или ще загубите баланса си от %s.</string>
+    <string name="notification_inactivity_message">Все още имате Dash на това устройство! Моля, не забравяйте да запишете фразата за възстановяване, преди да деинсталирате Dash Wallet или ще загубите баланса си от %s.</string>
```

Comment:
This type of formatting error (with an extra tab or spaces) comes from Transifex.  Other defects in strings will manifest themselves over time as well.

Even if we fix it in our code, Transifex will give it to us with the extra tab the next time we pull updates.  We would have to push all of our text to Transifex after the **redesign** is translated to resolve this issue.